plugins/server-features/ssr-page.ts

```
@@ -0,0 +1,173 @@
+        HTMLJSXWrapper(preBuiledPage)
+    );
+}
+
+function ErrorOnNoServerSideMatch(manager: RequestManager) {
+    return new Error(`no serverSideScript found for ${manager.pathname}`);
+}
+
+function HTMLJSXWrapper(html: string) {
+    return createElement("div", {
+        id: "BUNEXT_INNER_PAGE_INSERTER",
+        dangerouslySetInnerHTML: { __html: html },
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)